snippets/logging-support-exception-for-cluster-observability-operator-due-to-logging-ui-plugin.adoc

```diff
@@ -0,0 +1,10 @@
+// Text snippet included in the following assembly:
+//
+// * observability/logging/logging-6.0/log6x-visual.adoc
+
+:_mod-docs-content-type: SNIPPET
+
+[IMPORTANT]
+====
+Until the approaching general availability of the Cluster Observability Operator (COO), which is currently a link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview], Red Hat provides support to customers who are using Logging 6.0 or later with the COO for its xref:../../../observability/cluster_observability_operator/ui_plugins/logging-ui-plugin.adoc#logging-ui-plugin[Logging UI Plugin] on {product-title} 4.16 or later. This support exception is temporary as the COO incorporates several independent features, some of which are still link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview] features, but the xref:../../../observability/cluster_observability_operator/ui_plugins/logging-ui-plugin.adoc#logging-ui-plugin[Logging UI Plugin] is ready for general availability.
```

Comment:
```suggestion
Until the General Availability (GA) of the Cluster Observability Operator (COO), which is currently in link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview] (TP), Red Hat supports customers using Logging 6.0 or later with the COO for its xref:../../../observability/cluster_observability_operator/ui_plugins/logging-ui-plugin.adoc#logging-ui-plugin[Logging UI Plugin] on {product-title} 4.16 or later. This support is temporary as the COO includes several TP features, but the Logging UI Plugin is ready for GA.
```
I observed use of General Availability (GA) and Technology Preview (TP) here: https://docs.openshift.com/container-platform/4.17/support/troubleshooting/troubleshooting-operating-system-issues.html#investigating-kernel-crashes

"The x86_64 architecture supports kdump in General Availability (GA) status, whereas other architectures support kdump in Technology Preview (TP) status."

Reply:
Thank you for your suggestions. I included some of these suggestions. The other words I didn't change because this particular wording came out of the discussions that I had with Support yesterday.